### PR TITLE
Test for empty username

### PR DESCRIPTION
**Pull request by Errora**
Test for empty username

Close https://github.com/tarantool/cartridge/issues/779

### CODE REVIEW

File: webui/cypress/integration/login-and-logout.spec.js [hunk 1]
```diff
@@ -1,34 +1,70 @@
-//Steps:
-//1. Login error
-//2. Success login
+describe('Login', () => {
 
+  // it('Login error', () => {
```
Comment (rosik) [inline webui/cypress/integration/login-and-logout.spec.js:3]: Why do you comment it out?

File: webui/cypress/integration/remove-user.spec.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ describe('Remove user', () => {
     cy.get('a[href="/admin/cluster/users"]').click();
     cy.get('.meta-test__UsersTable').find('button').eq(1).click();
     cy.get('li').contains('Remove user').click();
-    cy.get('.meta-test__UserRemoveModal button[type="button"]').contains('Remove').click();
+    cy.get('.meta-test__UserRemoveModal button[type="button"]:contains(Remove)').click();
+    cy.reload();
```
Comment (rosik) [inline webui/cypress/integration/remove-user.spec.js:13]: Why? cy.reload is rather expensive. I didn't notice any problems before. Is it necessary?

File: webui/cypress/integration/uninitialized.spec.js [hunk 3]
```diff
@@ -1,14 +1,6 @@
 describe('Schema section', () => {
   it('Schema without bootstrap', () => {
     cy.visit(Cypress.config('baseUrl')+"/admin/cluster/schema");
-
-    cy.get('button[type="button"]:contains("Validate")').click();
-    cy.get('#root').contains('Cluster isn\'t bootstrapped yet');
```
Comment (rosik) [inline webui/cypress/integration/uninitialized.spec.js:6]: Why? It's not related to the topic of PR

File: webui/cypress/integration/login-and-logout.spec.js (hunk 1 above)
Comment (Errora) [reply webui/cypress/integration/login-and-logout.spec.js:3]: My mistake. Redid

File: webui/cypress/integration/remove-user.spec.js (hunk 2 above)
Comment (Errora) [reply webui/cypress/integration/remove-user.spec.js]: Test fell, i try again without reload

Comment (Errora) [reply webui/cypress/integration/remove-user.spec.js]: Сhanged reload for a short pause: 
cy.get('.meta-test__addUserBtn');

File: webui/cypress/integration/uninitialized.spec.js (hunk 3 above)
Comment (Errora) [reply webui/cypress/integration/uninitialized.spec.js:6]: Changed

**Pull request merged**